lib/widgets/nine_tile_box.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:flutter/widgets.dart' as widgets;
+import 'package:meta/meta.dart';
+import '../sprite.dart';
+import 'dart:ui';
+
+class _Painter extends widgets.CustomPainter {
+  final Image image;
+  final double tileSize;
+  final double destTileSize;
+
+  _Painter(
+      {@required this.image,
```

Comment:
can we have this formatted like so?
```
_Painter({
    @required ...,
    @required ...,
    @required ...,
});
```

Reply:
same thing as above, `flutter format` result

Reply:
because you lack trailing comma, please take a look again at my suggestion, it is totally formatter valid.